ui/src/contexts/Application.jsx

```diff
@@ -179,7 +180,14 @@ const watchCollateral = async (dispatch, collateral, treasuryAPI) => {
   const notifier = await E(manager).getNotifier();
   for await (const state of iterateNotifier(notifier)) {
     console.log('asset state', collateral.brand, state);
-    dispatch(mergeVaultAssets([[collateral.brand, state]]));
+    dispatch(
+      mergeVaultAssets([
+        [
+          collateral.brand,
+          { totalDebt: AmountMath.makeEmpty(runBrand), ...state },
```

Comment:
since you're the only one working on this code you're unlikely to forget, but generally it's best to mark intentional tech debt. I use `XXX` to note something that's subpar but can be left. `TODO` for something that works okay but should be improved. `FIXME` for something knowingly broken. I think that applies here. 

Also referencing the blocker issue helps people know _when_ it will be done and why not already.

Reply:
Done. Thanks for review!